docs/docs/configuration/alpha_config.md

```diff
@@ -377,6 +399,16 @@ as well as an optional minimal TLS version that is acceptable.
 | `Cert` | _[SecretSource](#secretsource)_ | Cert is the TLS certificate data to use.<br/>Typically this will come from a file. |
 | `MinVersion` | _string_ | MinVersion is the minimal TLS version that is acceptable.<br/>E.g. Set to "TLS1.3" to select TLS version 1.3 |
 
+### URLParameterRegexp
+
+(**Appears on:** [LoginURLParameter](#loginurlparameter))
+
+
+
+| Field | Type | Description |
+| ----- | ---- | ----------- |
+| `re` | _regexp.Regexp_ |  |
```

Comment:
Might need to teach the docs generator to not include private fields 🤔

Reply:
This has been rendered irrelevant with my latest revision.